.github/workflows/CHECK_UPDATE.yml

```diff
@@ -0,0 +1,43 @@
+name: CHECK_UPDATE
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '17 2 * * 0' # Every Sunday at 2:17 AM (random time to distribute load)
```

Comment:
I'd rather trigger it outside of the weekend. Let's use a meaningful value, e.g. third Tuesday of a month (middle of the release cycle).

Reply:
OK it looks cron can't handle that, but still workdays are when I'd expect PR.

Reply:
Fixed via https://github.com/camunda/feel-builtins/pull/4/commits/3019eccf76bddcc7fc057069ea65da358259037d